ansible/vm-update.yml

```diff
@@ -0,0 +1,77 @@
+---
+- name: preperations and sanity checks
+  hosts: "{{ hostname }}"
+  gather_facts: no
+  tasks:
+  - name: setup variables
+    set_fact:
+      network_cooked: "{{ network }}"
+      install_cooked: "{{ install }}"
+
+  - name: create temporary host group for vm host
+    changed_when: no
+    add_host:
+      name: "{{ install_cooked.host }}"
+      inventory_dir: "{{inventory_dir}}"
+      group: _vmhost_
+  # TODO: add some sanity checks
+
+- name: Update network configuration & shutdown VM
+  hosts: "{{ hostname }}"
+  gather_facts: no
+  roles:
+    - vm/network
+
+  tasks:
+    - meta: flush_handlers
+
+    - name: Shutdown VM
+      async: 1
+      poll: 0
+      ignore_errors: true
+      shell: sleep 2 && poweroff
+
+- name: Update VM definition
+  hosts: _vmhost_
+  vars:
+    run_installer: no
```

Comment:
isn't `run_installer: no` the default anyway? If not we should make it the default and omit this here

Reply:
`roles/vm/install` doesn't specify a default, fixing that in a sec.

Reply:
ok, thanks!